powerlevel9k.zsh-theme

```diff
@@ -973,18 +973,31 @@ build_right_prompt() {
     # Check if it is a custom command, otherwise interpet it as
     # a prompt.
     if [[ $element[0,7] =~ "custom_" ]]; then
-      "prompt_custom" "right" "$index" $element[8,-1]
+      "prompt_custom" "right" "$index" $element[8,-1] >> $POWERLEVEL9K_SOCKET
     else
-      "prompt_$element" "right" "$index"
+      "prompt_$element" "right" "$index" >> $POWERLEVEL9K_SOCKET
```

Comment:
How is `$POWERLEVEL9K_SOCKET` defined? I don't see it anywhere, but maybe I'm just missing it?